.applier/inventory/group_vars/all.yml

```diff
@@ -0,0 +1,15 @@
+openshift_cluster_content:
+- object: omp-frontend
+  content:
+  - name: imagestream
+    file: '{{ inventory_dir }}/../files/omp-frontend-imagestream.yml'
+  - name: buildconfig
+    template: '{{ inventory_dir }}/../templates/omp-frontend-buildconfig.yml'
+    params: '{{ inventory_dir }}/../params/omp-frontend-buildconfig'
+  - name: deploymentconfig
+    template: '{{ inventory_dir }}/../templates/omp-frontend-deploymentconfig.yml'
+    params: '{{ inventory_dir }}/../params/omp-frontend-deploymentconfig'
```

Comment:
Generally we show a preference for Params  forms cars. For me I like having them in your file but I’m ok with this approach